src/test/java/com/sngular/kloadgen/extractor/extractors/avro/AvroConfluentExtractorTest.java

```diff
@@ -1,30 +1,29 @@
-package com.sngular.kloadgen.extractor.extractors;
+package com.sngular.kloadgen.extractor.extractors.avro;
 
 import java.util.List;
 
 import com.sngular.kloadgen.model.FieldValueMapping;
 import com.sngular.kloadgen.testutil.FileHelper;
-import io.confluent.kafka.schemaregistry.ParsedSchema;
-import org.apache.avro.Schema;
+import io.confluent.kafka.schemaregistry.avro.AvroSchema;
 import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-class AvroExtractorTest {
+class AvroConfluentExtractorTest {
+
 
   private final FileHelper fileHelper = new FileHelper();
 
-  private final AvroExtractor avroExtractor = new AvroExtractor();
+  private final AvroConfluentExtractor avroConfluentExtractor = new AvroConfluentExtractor();
```

Comment:
I'll recommend here use the Interface as a Type. In case there were changes in the interface and some decided the remove the "implements" by mistake. There will be a chance to catch it here